### PR TITLE
Update clusterLanding.html

### PR DESCRIPTION
**Pull request by mokalebo**
Using Bootstrap form control to modify previous search box

### CODE REVIEW

**Comment by djw8605** [comment]
This is how it works for me.  The search box should be smaller, the width should be at most 1/3 of the width of the top.
image

**Comment by mokalebo** [comment]
I made some additional changes to the initial request

**Comment by djw8605** [comment]
It looks like it's screwing up the formatting:
image

File: HCCGo/app/html/clusterLanding.html [hunk 1]
```diff
@@ -54,7 +54,9 @@ <h4 class="usage text-center">/work Usage</h4>
 
     <div class="col-md-8 jobdisplay">
       <div class="card">
-        <input type="search" ng-model="q" placeholder="filter jobs..." />
+        <div class="col-xs-6">
+          <input type="search" class="form-control" placeholder="filter jobs..." />
```
Comment (djw8605) [inline HCCGo/app/html/clusterLanding.html]: I think you still need the `ng-model="q"` here.

File: HCCGo/app/html/clusterLanding.html [hunk 2]
```diff
@@ -85,6 +87,7 @@ <h4 class="list-group-item-heading">{{job.jobName}}</h4>
           <div class="list-group-separator" ng-repeat-end ng-if="!$last"></div>
         </div>
       </div>
+      </div>
```
Comment (djw8605) [inline HCCGo/app/html/clusterLanding.html]: Why is this div way down here?  It should have ended much higher!

**Pull request closed without merging**